README.md

```diff
@@ -1,95 +1,167 @@
-# eProsima Fast RTPS Shapes Demo
+# eProsima Fast DDS Shapes Demo
 
-The Shapes Demo is an application that Publishes and Subscribes to shapes of different colors and sizes moving on a board. Each Shape conforms its own topic: Square, Triangle or Circle. A single instance of the Shapes Demo can publish on or subscribe to several topics at a time.
+eProsima Shapes Demo is an application in which Publishers and Subscribers are shapes of different colors and sizes
+moving on a board.
+Each shape refers to its own topic: Square, Triangle or Circle.
+A single instance of the eProsima Shapes Demo can publish on or subscribe to several topics at a time.
 
-It can be used to demonstrate the capabilities of eProsima Fast RTPS or as an interoperability demonstrator with other implementations of the RTPS protocol.
+It can be used to demonstrate the capabilities of eProsima *Fast DDS* or as a proof of interoperability with other
+RTPS-compliant implementations.
```

Comment:
It'd be cool to see one of those images from the documentation repo here as well